text/0025-purchase-order.md

```diff
@@ -309,31 +308,30 @@ Beginning state. An organization with po::draft permissions created a version
 for collaboration purposes.
 
 Editable state. The contents of the version are being populated by one or more
-organizations. Any organization has permission to submit the draft for review
-or close it in favor of a better version.
+organizations. Any organization with po::draft permissions can submit the draft for review
+or close it in favor of a different version.
 
 Review state. An organization has proposed the version for review by another
 organization. Another organization has permission to approve the version by
-moving it to Composed state. The organization may also decline the version by
-adding a rejection reason (optional) or move it back to Editable state for
+moving it to Composed state. The organization may also decline the version,
+with the option to add a rejection reason, or move it back to Editable state for
 further modification.
 
-Composed state. Both organizations are in agreement on the contents of the
-version. The buying organization retains the decision to issue a formal
+Composed state. Both organizations have accepted the contents of the
+version. The buying organization may decide to issue a formal
 purchase order based on the drafted contents.
 
-Declined state. An organization has declined the version content. An
+Declined state. An organization has declined the version content. Any
```

Comment:
Hm, the use of "any" here makes it sound as though no permissions exist for that action